rust/windows-client/src-tauri/src/client/gui/system_tray_menu.rs

```diff
@@ -66,6 +66,16 @@ pub(crate) fn signed_in(user_name: &str, resources: &[ResourceDescription]) -> S
     menu
 }
 
+pub(crate) fn signing_in() -> SystemTrayMenu {
+    // TODO: Check this layout with Jamil, I lost track of the Figma for it.
```

Comment:
Post a screenshot? I'm not sure if we had one for "signing in" state

Reply:
![image](https://github.com/firezone/firezone/assets/13400041/41a6c234-8607-4e1d-84d6-06e506596442)

Reply:
Lgtm